woocommerce/woocommerce
# Product Gallery: Add Group block labels

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR adds Group block labels to the Product Gallery group inner blocks.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Appearance > Editor > Templates > Single Product (clear customizations)
2. Click on the editor and you should see "Transform into blocks" action button.
3. Add the Product Gallery block.
4. Open the sidebar navigation, verify that the Group block labels are the following:
- Gallery Area
- Large Image and Navigation



**Screenshot:**

<img width="347" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce/assets/905781/8b17482b-605b-45d1-9a92-d899668f7fd3">


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
